_posts/2016/08/2016-08-25-what-i-didnt-learn.html

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "What I Didn't Learn in a CS Degree"
+date: 2016-08-25 05:30
+---
+<p>
+  I recently stumbled across
+  <em><a href="https://bigmachine.io/imposter/">The Imposter's Handbook</a></em>,
+  which describes itself this way:
+</p>
+<blockquote>
+  <p>
+    For the longest time I would remain silent when discussions with
+    my peers would veer toward theoretical topics like P vs. NP,
+    Lambda Calculus or bubble sort vs. merge sort...  I decided to
+    change all of this a year ago. I sat down and looked up all of the
+    topics that a typical CS degree covers and then I dove in. Half
+    way through, I decided to write a book about what I was learning.
+  </p>
+</blockquote>
+<p>
+  It's an interesting project,
+  but it got me thinking about all the things I <em>didn't</em> learn
+  when I <em>did</em> do a CS degree:
+</p>
+<ul>
+  <li>
+    Most of what I've "accomplished" should actually be attributed
+    to being born white and male without serious mental or physical challenges
+    in a stable middle-class family
+    living in a peaceful democracy
+    at a time when we didn't yet have to acknowledge the long-term cost of our profligacy.
+    In short,
+    how well I played my cards is line noise compared to how lucky a hand I was dealt
+    compared to 95% of my fellow human beings.
+  </li>
+  <li>
+    The computing industry is actively unwelcoming to people who aren't straight white/Asian males,
+    routinely builds systems that facilitate their harassment and abuse,
+    and then <a href="https://twitter.com/mhoye/status/763955787869753344">pretends that the problem is somehow insoluble</a>.
```

Comment:
The tweet you linked doesn't seem to be about the computing industry... Although, I don't know who these people are...